lavamoat/browserify/policy-override.json

```diff
@@ -121,6 +121,12 @@
         "crypto.getRandomValues": true
       }
     },
+    "@metamask/notification-services-controller>firebase>@firebase/messaging": {
+      "globals": {
+        "PushManager.getSubscription": true,
+        "PushManager.subscribe": true
+      }
+    },
```

Comment:
Minimal scope used by Firebase for supporting push notifications.

More details can be found [here](https://github.com/search?q=repo%3Afirebase%2Ffirebase-js-sdk%20PushManager&type=code)